LFS171x/fabric-material/tuna-app/startFabric.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Exit on first error
+set -e
+
+# don't rewrite paths for Windows Git Bash users
+export MSYS_NO_PATHCONV=1
+
+starttime=$(date +%s)
+
+if [ ! -d ~/.hfc-key-store/ ]; then
+	mkdir ~/.hfc-key-store/
+fi
+#cp $PWD/creds/* ~/.hfc-key-store/
+# launch network; create channel and join peer to channel
+cd ../basic-network
```

Comment:
Would like to use the files from the fabric-samples repo instead of copying these here. Can we change the script to pull the fabric-samples repo or set an environment variable for when the repo was cloned in the previous step.